index.html

```diff
@@ -1,203 +1,171 @@
+                            <ul class="can">
+                            <li class="can"><a href=""><img class="twitter" src="twitter.png" alt="twitter"></a></li>
+                            <li class="can"><a href=""><img class="in" src="in.png" alt="in"></a></li>
+                            <li class="can"><a href=""><img class="medium" src="medium.png" alt="medium"></a></li>
+                            <li class="can"><a href=""><img class="github" src="github.png" alt="github"></a></li>
+                            <li class="can"><a href=""><img class="angellist" src="angellist.png" alt="angellist"></a></li></ul></div>
+                            <button class="get">Get my resume</button>
+                    </div>
+                    <div class="parttwo">
+                            <ul class="down">
+                                <li class="down"><div class="ic"><a class="lang">Languages</a><img class="vec" src="vect.png"></div></li>
+                                <ul class="ic">
+                                <li class="ic"><a class="js">JavaScript</a></li>
+                                <li class="ic"><a class="html5">HTML</a></li>
+                                <li class="ic"><a class="css3">CSS</a></li></ul>
+                                <li class="down"><div class="ic"><a class="lang">Frameworks</a><img class="vec-1" src="vect1.png"></div></li>
+                                <li class="down"><div class="ic"><img class="line" src="line.png"></div></li>
+                                <li class="down"><div class="ic"><a class="lang">Skills</a><img class="vec-1" src="vect1.png"></div></li>
+                                <li class="down"><div class="ic"><img class="line" src="line.png"></div></li>
+                            </ul>
+                    </div>
+                </article>
+                <article class="contactp">
+                    <div class="back">
+                    <div class="topback"><h1 class="contact-me">Contact me</h1>
+                    <p class="contact-page">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. 
+                        I’d love to help with it!</p></div>
+                    <div class="bottomback">
+                    <form action="https://formspree.io/f/xeqwyeqe" method="post">
+                        <input id="name" type="text" name="name" maxlength="30" minlength="3" placeholder="name" required pattern="[A-Z]*[a-z]*">
+                        <input id="email" type="email" name="email" placeholder="Email" required>
+                        <textarea id="message" maxlength="500" name="message" placeholder="Write your message here..." required></textarea>
+                        <button class="touch" type="submit">Get in touch</button>
+                        </form></div>
+                        <div class="foot"></div></div>
+                </article></article>

```

Comment:
- [ ] [OPTIONAL] Even though this seems to have been missed by the reviewer @AbelG101 , it is still a key feature which I strongly recommend that you make changes to as well. Your foter section has some styling differences to the one provided in the Figma design. The slight inconsistency in design is making your footer section to not look well styled and might impact negatively your user's user interface design.

**This is how your footer section looks like on my end:**

[Screencast from 14-06-2023 10:38:21 ALASIRI.webm](https://github.com/feyisakedir/Portfolio-2nd-Project/assets/86318284/2ec542c2-c221-475f-9b6f-a7fc12f47cb5)

**This is the desired Figma design:**

![image](https://github.com/feyisakedir/Portfolio-2nd-Project/assets/86318284/11939d12-efe9-4da7-bd51-206d4c5c0aac)

**Note:** making this change will improve the user interface and experience of your website and will help so much to make your website look professional and world-class.